src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-number;
+  flex-wrap: wrap;
+  display: flex;
+  width: $my-var;
```

Comment:
The variable `$my-var` is not descriptive enough. Consider renaming it to something more meaningful, like `$calendar-width`.